.circleci/config.yml

```diff
@@ -4,12 +4,17 @@ orbs:
   aws-s3:    circleci/aws-s3@2.0.0
   terraform: circleci/terraform@2.1.0
 
+parameters:
+  cross-container-tag:
+    type: string
+    default: go1.18.5-81cd97477f9a6c9f2db03dc56f560bf54f8da8bb
```

Comment:
Why are we changing this?

Reply:
Ah, I need to rebase from master and force push. Those extra commits are due to the history rewrite